cmd/otelcol/config/collector/full_config_linux.yaml

```diff
@@ -48,7 +48,7 @@ receivers:
   #  - grpc (default endpoint = 0.0.0.0:14250)
   #  - thrift_binary (default endpoint = 0.0.0.0:6832/udp)
   #  - thrift_compact (default endpoint = 0.0.0.0:6831/udp)
-  #  - thrift_http (default endpoint = 0.0.0.0:14268)
+  #  - thrift_http (default endpoint = 0.0.0.0:14268/api/traces)
```

Comment:
I believe the `/udp` designators above are to document that the receiver's udp socket is bound to the port (by convention notated w/ `6831/udp` vs `6831/tcp`).  These `endpoint` options are significant in the Collector and in this case shouldn't contain the actual url path: https://github.com/open-telemetry/opentelemetry-collector/blob/6458914bbd0d06bc3fa6304197fe736d47b0f694/receiver/jaegerreceiver/factory.go#L44.

We could provide all the desired paths as used by tracers/clients as part of https://github.com/signalfx/splunk-otel-collector/pull/301 or in another PR in documentation, unless this is something you'd be interested in doing here.

(cc @dmitryax)